src/routes/command/deleteMulti.ts

```diff
@@ -0,0 +1,36 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithClientAndSupplier } from '../../comms/controller';
+
+interface Request {
+  body: {
+    hashIds: string[];
+  };
+}
+
+type Response = Array<{
+  hashId: string,
+  deleted: boolean
+  reason?: string,
+}>;
+
+const controllerGeneratorOptions: ControllerGeneratorOptionsWithClientAndSupplier = {
+  method: 'delete',
+  path: '/',
+  body: Joi.object().keys({
+    hashIds: Joi.array().items(Joi.string()).required().example(['ga9741s']),
+  }).required(),
+  response: Joi.array().items(Joi.object().keys({
+    hashId: Joi.string().required().example('ga9741s'),
+    deleted: Joi.boolean().required().example(false),
+    reason: Joi.string().example('Device cannot be reached in time to cancel this command.'),
```

Comment:
Will we ever send a reason with a deleted:true? So is {hashId: 'ga9741s', deleted: true, reason: 'you requested the deletion'} something the server will answer. You could consider naming the reason reasonFailure

Reply:
Yes, I guess now that I'm sure this signature works, I could even go further and do a Union type, smth like:
```ts
type ValidResponse = {
  deleted: boolean = true
  hashId: string
}
type ErrorResponse = {
  deleted: boolean = false
  hashId: string
  failureReason: string
}
type Response = Array<ValidResponse | ErrorResponse>
```

(I don't remember if the boolean default is allowed in a type, but you get the idea I think)

And update the Joi schema accordingly

What do you think ?

Reply:
I would like it better. But not a hard requirement

Reply:
4cec2ac6b23c8cadf045c089a67f89cc30f852a5